lgrcommands/render.py

```diff
@@ -0,0 +1,68 @@
+"""
+
+"""
+import re
+import urllib.parse
+
+from pycldf import orm
+from jinja2 import Template
+
+from cldfbench_lgr import Dataset
+
+MD_LINK_PATTERN = re.compile(r'\[(?P<label>[^]]+)]\((?P<url>[^)]+)\)')
+
+def render_example(e):
+    print(e.cldf)
+    template = Template("""
+<blockquote>
+({{ example.id }}) {{ example.related('languageReference').name }}{% if example.references %}
+ ([{{ example.references[0].source.id }}](#source-{{ example.references[0].source.id }}){% if example.references[0].description %}
+: {{ example.references[0].description }}{% endif %}){% endif %}
+
+
+{% for word in example.cldf.analyzedWord %}{% if loop.first != True %} | {% endif%}{{ word }}{% endfor %}
+
+{% for word in example.cldf.analyzedWord %}{% if not loop.first %}|{% endif%} --- {% endfor %}
+
+{% for word in example.cldf.gloss %}{% if not loop.first %} | {% endif%}{{ word }}{% endfor %}
+
+
+‘{{ example.cldf.translatedText }}’
+</blockquote>
```

Comment:
@fmatter So this is how a template for a particular object type would look like. Since it receives a `pycldf.orm.Object`, accessing stuff like `references` and related objects is easy - if lengthy. Using Jinja2, it should be possible to provide sensible default templates, but give the end-user a way to override.